client/my-sites/plans/jetpack-plans/selector.tsx

```diff
@@ -132,7 +140,7 @@ const SelectorPageAlt: React.FC< SelectorPageProps > = ( {
 	const viewTrackerProps = siteId ? { site: siteSlug } : {};
 
 	return (
-		<Main className="selector-alt__main" wideLayout>
```

Comment:
Searching the codebase for `.selector-alt__main`, it looks like there are some e2e tests (and 1 .scss file) that are still referencing this `.selector-alt__main` class.  They'll need to be updated to reference `.selector__main` instead.
Files:
`client/jetpack-connect/style.scss`
`test/e2e/lib/pages/plans-page.js`
`test/e2e/lib/pages/external/jetpackcom-pricing-page.js`
`test/e2e/lib/pages/signup/pick-a-plan-page.js`

Reply:
Good catch, thanks @elliottprogrammer! I pushed a new commit.